README.md

```diff
@@ -2,48 +2,107 @@
+
+1. [Manual clone & build](#manual-clone-&-build)
+2. [Go](#go)
+3. [RPM](#rpm)
+
+
+### Manual clone & build
+
+```
+git clone https://github.com/ClusterLabs/ha_cluster_exporter
+cd ha_cluster_exporter
+make
+make install
+```
+
+### Go
+
+```
+go get github.com/ClusterLabs/ha_cluster_exporter
+```
+
+### RPM
+You can find the repositories for RPM based distributions in [SUSE's Open Build Service](https://build.opensuse.org/repositories/server:monitoring/prometheus-ha_cluster_exporter).  
+On openSUSE or SUSE Linux Enterprise you can just use the `zypper` system package manager:
+```shell
+export DISTRO=SLE_15_SP1 # change as desired
+zypper addrepo https://download.opensuse.org/repositories/server:/monitoring/$DISTRO/server:monitoring.repo
+zypper install prometheus-ha_cluster_exporter
+```
+
+## Usage
+
+You can run the exporter in any of the cluster nodes. 
+
+```
+$ ./ha_cluster_exporter  
+INFO[0000] Serving metrics on 0.0.0.0:9002
+```
+
+Though not strictly required, it is _strongly_ advised to run it in all the nodes.
+
+It will export the metrics under the `/metrics` path, on port `9002` by default.
+
+While the exporter can run outside a HA cluster node, it won't export any metric it can't collect; e.g. it won't export DRBD metrics if it can't be locally inspected with `drbdsetup`.  
+A warning message will inform the user of such cases.
 
 **Hint:**
-For a terraform deployment you can also read: https://github.com/SUSE/ha-sap-terraform-deployments
+You can deploy a full HA Cluster via Terraform with [SUSE/ha-sap-terraform-deployments](https://github.com/SUSE/ha-sap-terraform-deployments).
+
+### Configuration
+
+All the runtime parameters can be configured either via CLI flags or via a configuration file, both or which are completely optional.
+
+For more details, refer to the help message via `ha_cluster_exporter --help`.
+
+**Note**:
+the built-in defaults are tailored for the latest version of SUSE Linux Enterprise.
 
-## Features:
+The program will scan, in order, the current working directory, `$HOME/.config`, `/etc` and `/usr/etc` for files named `ha_cluster_exporter.(yaml|json|toml)`.
+The first match has precedence, and the CLI flags have precedence over the config file.
 
-- show cluster node and resource metrics via `crm_mon` (pacemaker data xml)
+Please refer to the example [YAML configuration](ha_cluster_exporter.yaml) for more details.
 
-- show corosync metrics (ring errors, quorum metrics)
+### systemd integration
 
-- show SBD disk health metrics
+A [systemd unit file](prometheus-ha_cluster_exporter.service) is provided with the RPM packages. You can enable and start it as usual:  
 
-- show DRBD metrics (local and remote disks resource metrics)
+```
+systemctl --now enable prometheus-ha_cluster_exporter
+```
 
-We mantain a complete list of the [metric specification](doc/metric_spec.md), usage and possible values.
+## Development
 
-## Devel:
+This is a young project, pull requests are more than welcome!
```

Comment:
I'm not really in favor to keep this. This is a young project,
I mean from experience in future we will forget to remove this string and it will stay forever young .. I wish also :grin: